static/js/components/search/CountryRefinementOption.js

```diff
@@ -0,0 +1,40 @@
+// @flow
```

Comment:
i was going to say that this component should be more generalized, but i guess we can take care of that as we add new facets. just something to keep in mind for future work on the search page